include/openflow/orange-ext.h

```diff
@@ -53,4 +53,36 @@ struct ol_update_map {
                                    entries (key + value). */
 };
 OFP_ASSERT(sizeof(struct ol_update_map) == 16);
+
+struct ol_dump_map_request {
+    ovs_be16 filter_prog;  /* Filter program ID. */
+    ovs_be16 nb_maps;
+    /* Followed by:
+     *   - Exactly nb_maps map_ids of total nb_maps * ovs_be16 bytes. */
+    /* uint8_t entries[...]; */ /* Data containing the map ids. */
+};
+OFP_ASSERT(sizeof(struct ol_dump_map_request) == 4);
+
+struct ol_dump_map_reply {
+    ovs_be16 filter_prog;  /* Filter program ID. */
+    ovs_be16 nb_maps;
+    /* Followed by:
+     *   - Exactly nb_maps ol_dump_maps of total
+     *     nb_maps * ol_dump_map bytes. */
+    /* uint8_t entries[...]; */ /* Data containing the ol_dump_maps. */
+};
+OFP_ASSERT(sizeof(struct ol_dump_map_reply) == 4);
+
+struct ol_dump_map {
+    ovs_be16 map_id;  /* Map ID. */
```

Comment:
I think it should just be `map`, to be consistent with `filter_prog` above.

Reply:
Done.